src/utils.js

```diff
@@ -28,6 +28,7 @@ function isEmptyObject(what) {
 function isError(what) {
     return isObject(what) &&
         objectPrototype.toString.call(what) === '[object Error]' ||
+        objectPrototype.toString.call(what) === '[object Exception]' || // Firefox NS_ERROR_FAILURE Exceptions
```

Comment:
Why not just do:

```javascript
['[object Error]', '[object Exception]'].indexOf(objectPrototype.toString.call(what)) > -1
```

Then we only need to `toString()` once and if we need to add to this, just add to this array. This array could also be popped out into a constant for like, `var ERROR_TYPES = ['[object Error]', '[object Exception]']`

Other than this, seems reasonable.